cypress/integration/spec-sidebar.ts

```diff
@@ -41,6 +41,12 @@ describe('The Sidebar', () => {
       cy.getIframeContent('plugin', 'window').its('__messageReceived__').should('eq', messageToSend)
     })
 
+    // To activate when button is available on the app
+    // it('should be notified when the minimize button is clicked', function () {
+    //   cy.get('[data-testid="sidebar-header-minimize"]').click()
+    //   cy.getIframeContent('plugin', 'window').its('__onMinimizeTriggered__').should('eq', true)
+    // })
+
     describe('on close', () => {
 
```

Comment:
The test doesn't pass ?

Reply:
Yes, since the button is new in the app, the test doesn't pass.